osmaxx-py/osmaxx/excerptexport/migrations/0008_auto_20150915_1417.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
see [Squashing Migrations](https://docs.djangoproject.com/en/1.8/topics/migrations/#squashing-migrations)

But I was wrong, I confused it with Rails, we should just keep it as is.